src/coreclr/jit/codegenarm64.cpp

```diff
@@ -6362,6 +6362,31 @@ void CodeGen::genArm64EmitterUnitTests()
     theEmitter->emitIns_R_R(INS_rev, EA_4BYTE, REG_R10, REG_R5);
     theEmitter->emitIns_R_R(INS_rev16, EA_4BYTE, REG_R11, REG_R6);
 
+    // TODO-SVE: Fix once we add predicate registers
+    // decp
+    theEmitter->emitIns_R_R(INS_sve_decp, EA_8BYTE, REG_R0, REG_R1, INS_OPTS_8B); // DECP    <Xdn>, <Pm>.<T>
```

Comment:
Temporarily we should probably add all the predicate registers to registerarm64.h but define them as equal to the vector register numbers, e.g.:
```
REGALIAS(REG_P0, REG_V0)
REGALIAS(REG_P1, REG_V1)
...
```

Then use `REG_P1` here instead of `REG_R1`

Reply:
`emitter::isPredicateRegister` could use these definitions.

Reply:
I think that's a good idea. Kunal has a couple of similar TODOs in #95016, so if this PR gets merged in first, maybe we should update his PR to use the aliases. @kunalspathak